src/analysis/normed_space/inner_product.lean

```diff
@@ -1575,33 +1575,52 @@ begin
+  simp,
+end
+
 /-- The orthogonal projection onto a complete subspace. -/
-def orthogonal_projection (K : submodule 𝕜 E) [complete_space K] : E →ₗ[𝕜] K :=
-{ to_fun := λ v, ⟨orthogonal_projection_fn K v, orthogonal_projection_fn_mem v⟩,
-  map_add' := λ x y, begin
-    have hm : orthogonal_projection_fn K x + orthogonal_projection_fn K y ∈ K :=
-      submodule.add_mem K (orthogonal_projection_fn_mem x) (orthogonal_projection_fn_mem y),
-    have ho :
-      ∀ w ∈ K, ⟪x + y - (orthogonal_projection_fn K x + orthogonal_projection_fn K y), w⟫ = 0,
-    { intros w hw,
-      rw [add_sub_comm, inner_add_left, orthogonal_projection_fn_inner_eq_zero _ w hw,
-          orthogonal_projection_fn_inner_eq_zero _ w hw, add_zero] },
-    ext,
-    simp [eq_orthogonal_projection_fn_of_mem_of_inner_eq_zero hm ho]
-  end,
-  map_smul' := λ c x, begin
-    have hm : c • orthogonal_projection_fn K x ∈ K :=
-      submodule.smul_mem K _ (orthogonal_projection_fn_mem x),
-    have ho : ∀ w ∈ K, ⟪c • x - c • orthogonal_projection_fn K x, w⟫ = 0,
-    { intros w hw,
-      rw [←smul_sub, inner_smul_left, orthogonal_projection_fn_inner_eq_zero _ w hw, mul_zero] },
-    ext,
-    simp [eq_orthogonal_projection_fn_of_mem_of_inner_eq_zero hm ho]
-  end }
+def orthogonal_projection (K : submodule 𝕜 E) [complete_space K] : E →L[𝕜] K :=
+linear_map.mk_continuous
+  { to_fun := λ v, ⟨orthogonal_projection_fn K v, orthogonal_projection_fn_mem v⟩,
+    map_add' := λ x y, begin
+      have hm : orthogonal_projection_fn K x + orthogonal_projection_fn K y ∈ K :=
+        submodule.add_mem K (orthogonal_projection_fn_mem x) (orthogonal_projection_fn_mem y),
+      have ho :
+        ∀ w ∈ K, ⟪x + y - (orthogonal_projection_fn K x + orthogonal_projection_fn K y), w⟫ = 0,
+      { intros w hw,
+        rw [add_sub_comm, inner_add_left, orthogonal_projection_fn_inner_eq_zero _ w hw,
+            orthogonal_projection_fn_inner_eq_zero _ w hw, add_zero] },
+      ext,
+      simp [eq_orthogonal_projection_fn_of_mem_of_inner_eq_zero hm ho]
+    end,
+    map_smul' := λ c x, begin
+      have hm : c • orthogonal_projection_fn K x ∈ K :=
+        submodule.smul_mem K _ (orthogonal_projection_fn_mem x),
+      have ho : ∀ w ∈ K, ⟪c • x - c • orthogonal_projection_fn K x, w⟫ = 0,
+      { intros w hw,
+        rw [←smul_sub, inner_smul_left, orthogonal_projection_fn_inner_eq_zero _ w hw, mul_zero] },
+      ext,
+      simp [eq_orthogonal_projection_fn_of_mem_of_inner_eq_zero hm ho]
+    end }
+  1
+  (λ x, begin
+    simp only [one_mul, linear_map.coe_mk],
+    refine le_of_pow_le_pow 2 (norm_nonneg _) (by norm_num) _,
+    change ∥orthogonal_projection_fn K x∥ ^ 2 ≤ ∥x∥ ^ 2,
+    nlinarith [orthogonal_projection_fn_norm_sq K x]
+  end)
 
 @[simp]
 lemma orthogonal_projection_fn_eq {K : submodule 𝕜 E} [complete_space K] (v : E) :
-  orthogonal_projection_fn K v = orthogonal_projection K v :=
+  orthogonal_projection_fn K v = ↑(orthogonal_projection K v) :=
```

Comment:
```suggestion
  orthogonal_projection_fn K v = (orthogonal_projection K v : E) :=
```
Actually I guess you could even write `orthogonal_projection K v` but the above version may be clearer.